additional.css

```diff
@@ -22,7 +22,7 @@ main {
     display: flex;
     justify-content: flex-start;
     align-items: center;
-    border-bottom: 1px solid #d0d9dd;
+    padding: 6px !important;
```

Comment:
The lack of padding was deliberate in this case. I did wonder why it didn't get a border, though... now I know :)